src/components/AppAdvert/AppAdvert.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './AppAdvert.scss';
+
+function AppAdvert() {
+  return (
+    <div className="app-advert">
+    </div>
```

Comment:
If you are not going to put anything inside the div:
```suggestion
    <div className="app-advert" />
```